### ISSUE TITLE
Lacking detail on different API versions

### ISSUE THREAD

**Issue by ThisIsRobokitty** [circleci/circleci-docs#1654]
This issue is about <https://circleci.com/docs/2.0/language-android/> (source file: <https://github.com/circleci/circleci-docs/blob/master/jekyll/_cci2/language-android.md>)

Where do I find out what other versions are available and how to get them on my CircleCI?

**Comment by felicianotech** [comment]
This list is available on the Android website/the CircleCI Docker images list. Maybe rewording is useful?
